Makefile.d/proto.mk

```diff
@@ -66,11 +67,17 @@ $(GOPATH)/src/github.com/protocolbuffers/protobuf:
 		https://github.com/protocolbuffers/protobuf \
 		$(GOPATH)/src/github.com/protocolbuffers/protobuf
 
-$(GOPATH)/src/github.com/googleapis/googleapis:
+$(GOPATH)/src/github.com/gogo/googleapis:
+	git clone \
+		--depth 1 \
+		https://github.com/gogo/googleapis \
+		$(GOPATH)/src/github.com/gogo/googleapis
```

Comment:
is this better than googleapis/googleapis?

Reply:
https://github.com/vdaas/vald/pull/934/files/159c6611ca5de18590705f874bf6f77457e285af#r559865248